lib/actions/bulkCreateObjects.js

```diff
@@ -0,0 +1,26 @@
+const { messages } = require('elasticio-node');
+const SugarCrm = require('../sugarcrm');
+
+exports.process = async function ProcessAction(msg, cfg) {
+  if (!msg.body.objects || !Array.isArray(msg.body.objects)) {
+    this.logger.error('input parameter "Objects" isn\'t an Array');
+    this.emit('data', messages.newMessageWithBody({ }));
```

Comment:
Why it doesn't throw an error (or emit error) instead of emitting data (empty object)?
Is it a specific behavior? Should it be documented additionally, because it is not a common-based approach for elastic.io-components.

Reply:
You're absolutely right. I must be looked into the wrong example.